I'm sorry, but an uncaught exception occurred.

<<<<<<< HEAD
While running game code:
  File "game/script.rpy", line 701, in script call
    call choix_retour_village_1 pass (massacre = True)
  File "game/script.rpy", line 915, in script call
    call massacre_foret_2 pass (lieu = "chateau")
  File "game/script.rpy", line 1131, in script call
    call attaque_massacre_foret_2 pass (message = "deception")
  File "game/script.rpy", line 1223, in script call
    call e_demande_nom_foret_2 pass (bad_ending = true)
  File "game/script.rpy", line 1230, in script
    label e_demande_nom_foret_2(bad_ending = False):
  File "game/script.rpy", line 1223, in <module>
    call e_demande_nom_foret_2 pass (bad_ending = true)
NameError: name 'true' is not defined
=======
While loading <'Image' 'forest_night.jpg'>:
  File "game/script.rpy", line 700, in script call
    call e_fouiller_village_1 pass (einarFouille = True)
  File "game/script.rpy", line 934, in script call
    call foret_2 pass (lieu = "nord")
  File "game/script.rpy", line 953, in script
    "En poursuivant vers le nord..."
IOError: Couldn't find file 'forest_night.jpg'.
>>>>>>> 2731dd90b59ca650ea12fee3263e1fad839f419c

-- Full Traceback ------------------------------------------------------------

Full traceback:
<<<<<<< HEAD
  File "game/script.rpy", line 701, in script call
    call choix_retour_village_1 pass (massacre = True)
  File "game/script.rpy", line 915, in script call
    call massacre_foret_2 pass (lieu = "chateau")
  File "game/script.rpy", line 1131, in script call
    call attaque_massacre_foret_2 pass (message = "deception")
  File "game/script.rpy", line 1223, in script call
    call e_demande_nom_foret_2 pass (bad_ending = true)
  File "game/script.rpy", line 1230, in script
    label e_demande_nom_foret_2(bad_ending = False):
  File "D:\Bibliothèque\Downloads\renpy-6.99.10-sdk\renpy\ast.py", line 1319, in execute
    args, kwargs = self.arguments.evaluate()
  File "D:\Bibliothèque\Downloads\renpy-6.99.10-sdk\renpy\ast.py", line 179, in evaluate
    kwargs[k] = renpy.python.py_eval(v, locals=scope)
  File "D:\Bibliothèque\Downloads\renpy-6.99.10-sdk\renpy\python.py", line 1606, in py_eval
    return py_eval_bytecode(code, globals, locals)
  File "D:\Bibliothèque\Downloads\renpy-6.99.10-sdk\renpy\python.py", line 1601, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/script.rpy", line 1223, in <module>
    call e_demande_nom_foret_2 pass (bad_ending = true)
NameError: name 'true' is not defined
=======
  File "game/script.rpy", line 700, in script call
    call e_fouiller_village_1 pass (einarFouille = True)
  File "game/script.rpy", line 934, in script call
    call foret_2 pass (lieu = "nord")
  File "game/script.rpy", line 953, in script
    "En poursuivant vers le nord..."
  File "D:\Logiciels\Renpy\renpy-6.99.10-sdk\renpy\ast.py", line 604, in execute
    renpy.exports.say(who, what, interact=self.interact)
  File "D:\Logiciels\Renpy\renpy-6.99.10-sdk\renpy\exports.py", line 1122, in say
    who(what, interact=interact)
  File "D:\Logiciels\Renpy\renpy-6.99.10-sdk\renpy\character.py", line 841, in __call__
    self.do_display(who, what, cb_args=self.cb_args, **display_args)
  File "D:\Logiciels\Renpy\renpy-6.99.10-sdk\renpy\character.py", line 690, in do_display
    **display_args)
  File "D:\Logiciels\Renpy\renpy-6.99.10-sdk\renpy\character.py", line 493, in display_say
    rv = renpy.ui.interact(mouse='say', type=type, roll_forward=roll_forward)
  File "D:\Logiciels\Renpy\renpy-6.99.10-sdk\renpy\ui.py", line 277, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "D:\Logiciels\Renpy\renpy-6.99.10-sdk\renpy\display\core.py", line 2425, in interact
    repeat, rv = self.interact_core(preloads=preloads, **kwargs)
  File "D:\Logiciels\Renpy\renpy-6.99.10-sdk\renpy\display\core.py", line 2779, in interact_core
    self.draw_screen(root_widget, fullscreen_video, (not fullscreen_video) or video_frame_drawn)
  File "D:\Logiciels\Renpy\renpy-6.99.10-sdk\renpy\display\core.py", line 1848, in draw_screen
    renpy.config.screen_height,
  File "renpy/display/render.pyx", line 416, in renpy.display.render.render_screen (gen\renpy.display.render.c:6685)
    rv = render(root, width, height, 0, 0)
  File "renpy/display/render.pyx", line 185, in renpy.display.render.render (gen\renpy.display.render.c:2857)
    rv = d.render(widtho, heighto, st, at)
  File "D:\Logiciels\Renpy\renpy-6.99.10-sdk\renpy\display\layout.py", line 649, in render
    surf = render(child, width, height, cst, cat)
  File "renpy/display/render.pyx", line 103, in renpy.display.render.render (gen\renpy.display.render.c:3319)
    cpdef render(d, object widtho, object heighto, double st, double at):
  File "renpy/display/render.pyx", line 185, in renpy.display.render.render (gen\renpy.display.render.c:2857)
    rv = d.render(widtho, heighto, st, at)
  File "D:\Logiciels\Renpy\renpy-6.99.10-sdk\renpy\display\layout.py", line 649, in render
    surf = render(child, width, height, cst, cat)
  File "renpy/display/render.pyx", line 103, in renpy.display.render.render (gen\renpy.display.render.c:3319)
    cpdef render(d, object widtho, object heighto, double st, double at):
  File "renpy/display/render.pyx", line 185, in renpy.display.render.render (gen\renpy.display.render.c:2857)
    rv = d.render(widtho, heighto, st, at)
  File "D:\Logiciels\Renpy\renpy-6.99.10-sdk\renpy\display\layout.py", line 649, in render
    surf = render(child, width, height, cst, cat)
  File "renpy/display/render.pyx", line 103, in renpy.display.render.render (gen\renpy.display.render.c:3319)
    cpdef render(d, object widtho, object heighto, double st, double at):
  File "renpy/display/render.pyx", line 185, in renpy.display.render.render (gen\renpy.display.render.c:2857)
    rv = d.render(widtho, heighto, st, at)
  File "renpy/display/accelerator.pyx", line 104, in renpy.display.accelerator.transform_render (gen\renpy.display.accelerator.c:1965)
    cr = render(child, widtho, heighto, st - self.child_st_base, at)
  File "renpy/display/render.pyx", line 185, in renpy.display.render.render (gen\renpy.display.render.c:2857)
    rv = d.render(widtho, heighto, st, at)
  File "D:\Logiciels\Renpy\renpy-6.99.10-sdk\renpy\display\image.py", line 261, in render
    return wrap_render(self.target, width, height, st, at)
  File "D:\Logiciels\Renpy\renpy-6.99.10-sdk\renpy\display\image.py", line 124, in wrap_render
    rend = render(child, w, h, st, at)
  File "renpy/display/render.pyx", line 103, in renpy.display.render.render (gen\renpy.display.render.c:3319)
    cpdef render(d, object widtho, object heighto, double st, double at):
  File "renpy/display/render.pyx", line 185, in renpy.display.render.render (gen\renpy.display.render.c:2857)
    rv = d.render(widtho, heighto, st, at)
  File "D:\Logiciels\Renpy\renpy-6.99.10-sdk\renpy\display\im.py", line 478, in render
    im = cache.get(self)
  File "D:\Logiciels\Renpy\renpy-6.99.10-sdk\renpy\display\im.py", line 200, in get
    surf = image.load()
  File "D:\Logiciels\Renpy\renpy-6.99.10-sdk\renpy\display\im.py", line 529, in load
    surf = renpy.display.pgrender.load_image(renpy.loader.load(self.filename), self.filename)
  File "D:\Logiciels\Renpy\renpy-6.99.10-sdk\renpy\loader.py", line 533, in load
    raise IOError("Couldn't find file '%s'." % name)
IOError: Couldn't find file 'forest_night.jpg'.
>>>>>>> 2731dd90b59ca650ea12fee3263e1fad839f419c

Windows-8-6.2.9200
Ren'Py 6.99.10.1227
Harald's Dusk 0.0
